nexmark-flink/src/main/java/com/github/nexmark/flink/metric/BenchmarkMetric.java

```diff
@@ -27,18 +27,28 @@
 public class BenchmarkMetric {
 	private final double tps;
 	private final double cpu;
+	private final boolean preciseMode;
 
 	public BenchmarkMetric(double tps, double cpu) {
+		this(tps, cpu, false);
+	}
+
+	public BenchmarkMetric(double tps, double cpu, boolean preciseMode) {
```

Comment:
I think `BenchmarkMetric` doesn't need `preciseMode`. It should carry the full information. 
Whether to print higher precision is decided by console (i.e. the benchmark client).